huaweicloud/services/dws/data_source_huaweicloud_dws_flavors.go

```diff
@@ -215,6 +222,17 @@ func flattenListNodeTypesFlavors(resp interface{}) []interface{} {
 	return rst
 }
 
+func obtainFlavorsDatastoreVersion(resp interface{}) string {
+	datastore, err := jmespath.Search("datastores[0]", resp)
```

Comment:
can we search the `version` directly?
```suggestion
	datastore, err := jmespath.Search("datastores[0].version", resp)
```